extensions/ql-vscode/src/remote-queries/sarif-processing.ts

```diff
@@ -15,6 +16,11 @@ import {
   HighlightedRegion,
 } from "./shared/analysis-result";
 
+// A line of more than 8k characters is probably generated.
+const CODE_SNIPPET_LARGE_LINE_SIZE_LIMIT = 8192;
+// If less than 1% of the line is highlighted, we consider it a small snippet.
+const CODE_SNIPPET_HIGHLIGHTED_REGION_MINIMUM_PERCENTAGE = 0.01;
```

Comment:
```suggestion
const CODE_SNIPPET_HIGHLIGHTED_REGION_MINIMUM_RATIO = 0.01;
```
or
```suggestion
const CODE_SNIPPET_HIGHLIGHTED_REGION_MINIMUM_PERCENTAGE = 1;
```

Calling this "percentage" when the value it contains is out of 1.0 rather than out of 100 could potentially cause confusion in the future to someone who just goes by the variable name. I don't have a preference between changing name or changing the value.